### ISSUE TITLE
Can't rebuild using tulun -r without also running a file

### ISSUE THREAD

**Issue by towerofnix** [liam4/tlnccuwagnf#24]
```
$ ./tulun.js
Usage: tulun <file>
--rebuild / -r : development only rebuilds source

$ ./tulun.js -r
-r.tul doesn't exist!

$ ./tulun.js -r test.tul
Building...
[program output]
```

**Comment by bates64** [comment]
On it :package:

**Comment by towerofnix** [comment]
Fixed in https://github.com/liam4/tlnccuwagnf/commit/6af8c2ff7db7d9d66a3bad4792a8971b0da2a336.

**Comment by towerofnix** [comment]
Btw -- you can put "fixes #24" in the message of the commit and it should automatically close the issue.

**Comment by bates64** [comment]
ah.
